pom.xml

```diff
@@ -83,6 +83,7 @@
     <java8.home>/Library/Java/JavaVirtualMachines/jdk1.8.0_261.jdk/Contents/Home</java8.home>
     <java11.home>/Library/Java/JavaVirtualMachines/jdk-11.0.8.jdk/Contents/Home</java11.home>
     <skip.forbidenApi>false</skip.forbidenApi>
+  	<closeTestReports>true</closeTestReports>
```

Comment:
I like this change, can we change this property name to surefire.disableXmlReport ? (it would be more consistent)